TheraLang.BLL/LiqPay/LiqPayHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Linq;
+using TheraLang.BLL.DataTransferObjects.Donations;
+
+namespace TheraLang.BLL.LiqPay
+{
+    public static class LiqPayHelper
+    {
+        public static string ConvertToQueryString<T>(this T obj)
+        {
+            return string.Join("&", obj.GetType()
```

Comment:
what about using string builder here?

Reply:
string.Join() uses string builder

Reply:
fair enough, also it's more readable and faster